src/libraries/Common/tests/TestUtilities/System/IO/PathGenerator.cs

```diff
@@ -17,6 +19,27 @@ public static string GenerateTestFileName([CallerMemberName] string memberName =
                 memberName ?? "TestBase",
                 lineNumber,
                 index.GetValueOrDefault(),
-                Guid.NewGuid().ToString("N").Substring(0, 8)); // randomness to avoid collisions between derived test classes using same base method concurrently
+                GenerateRandomFileSafeString(8)); // randomness to avoid collisions between derived test classes using same base method concurrently
+
+        private static string GenerateRandomFileSafeString(int length)
+        {
+            // A little more entropy than Guid.NewGuid().ToString("N").Substring(0, length))
+            const string alphanum = "abcdefghijklmnopqrstuvwxyz0123456789";
```

Comment:
IIRC `const`s are written in Pascal.
```suggestion
            const string Alphanumeric = "abcdefghijklmnopqrstuvwxyz0123456789";
```